example/lambdas/db-provision-user-database/tests/test-index.js

```diff
@@ -87,29 +93,82 @@ test('provision user fails if invalid user string is used', async (t) => {
   }));
 });
 
-test('provision user updates the user password if the user already exists', async (t) => {
+test('provision user database handler updates the user password', async (t) => {
   const {
     expectedDbUser,
     expectedTestDb,
     handlerEvent,
   } = t.context;
 
+  const knexConfig = {
+    client: 'pg',
+    connection: {
+      host: 'localhost',
+      user: expectedDbUser,
+      password: 'testPassword',
+      database: expectedTestDb,
+    },
+  };
+
   await handler(handlerEvent);
+  t.context.testKnex = Knex(knexConfig);
+  let heartBeat = await t.context.testKnex.raw('SELECT 1');
+  t.is(heartBeat.rowCount, 1);
+  t.context.testKnex.destroy();
```

Comment:
Does this need to be `await`ed throughout?

Reply:
That's probably good hygiene yes: fb5e0b545

Reply:
I think this hasn't been pushed up yet

Reply:
Github also doesn't retroactively apply the link : fb5e0b545